frontend/src/components/SideNav.tsx

```diff
@@ -257,302 +257,318 @@ export class SideNav extends React.Component<SideNavInternalProps, SideNavState>
-              title={`Build date: ${displayBuildInfo.date}, Commit hash: ${displayBuildInfo.commitHash}`}
-              enterDelay={300}
-              placement={'top-start'}
-            >
-              <div className={css.envMetadata}>
-                <span>Version: </span>
-                <a
-                  href={displayBuildInfo.commitUrl}
-                  className={classes(css.link, commonCss.unstyled)}
-                  rel='noopener'
-                  target='_blank'
+                  <div className={css.envMetadata}>
+                    <span>Cluster name: </span>
+                    <a
+                      href={`https://console.cloud.google.com/kubernetes/list?project=${gkeMetadata.projectId}&filter=name:${gkeMetadata.clusterName}`}
+                      className={classes(css.link, commonCss.unstyled)}
+                      rel='noopener'
+                      target='_blank'
+                    >
+                      {gkeMetadata.clusterName}
+                    </a>
+                  </div>
+                </Tooltip>
+              )}
+              {displayBuildInfo && (
+                <Tooltip
+                  title={`Build date: ${displayBuildInfo.date}, Commit hash: ${displayBuildInfo.commitHash}`}
+                  enterDelay={300}
+                  placement={'top-start'}
                 >
-                  {displayBuildInfo.tagName}
-                </a>
-              </div>
-            </Tooltip>
-          )}
-          <Tooltip title='Report an Issue' enterDelay={300} placement={'top-start'}>
-            <div className={css.envMetadata}>
-              <a
-                href={ExternalLinks.GITHUB_ISSUE}
-                className={classes(css.link, commonCss.unstyled)}
-                rel='noopener'
-                target='_blank'
-              >
-                Report an Issue
-              </a>
+                  <div className={css.envMetadata}>
+                    <span>Version: </span>
+                    <a
+                      href={displayBuildInfo.commitUrl}
+                      className={classes(css.link, commonCss.unstyled)}
+                      rel='noopener'
+                      target='_blank'
+                    >
+                      {displayBuildInfo.tagName}
+                    </a>
+                  </div>
+                </Tooltip>
+              )}
+              <Tooltip title='Report an Issue' enterDelay={300} placement={'top-start'}>
+                <div className={css.envMetadata}>
+                  <a
+                    href={ExternalLinks.GITHUB_ISSUE}
+                    className={classes(css.link, commonCss.unstyled)}
+                    rel='noopener'
+                    target='_blank'
+                  >
+                    Report an Issue
+                  </a>
+                </div>
+              </Tooltip>
             </div>
-          </Tooltip>
-        </div>
-      </div>
+          </div>
+        )}{' '}
```

Comment:
Question: Why do we add `{' '}` here?

Reply:
I must have left a whitespace between the parenthesis and `<>` before running prettier. Will fix it